index.d.ts

```diff
@@ -38,6 +38,12 @@ declare const clear: {
 	@param regex - Regex to match against the module IDs.
 	*/
 	match(regex: RegExp): void;
+
+	/**
+	Clear only one single module from cache
```

Comment:
I think it should clearly state, in English prose, not code example, what this method is for, with specific use-cases. I'm afraid people will use it by mistake and encounter weird issues later on when some sub-dependencies are cached.